specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -7760,12 +7769,13 @@
           "readOnly": true,
           "description": "Describes the reboot requirements of the patch.",
           "enum": [
+            "Unknown",
```

Comment:
Are the additions of all the 'Unknown' values to the enum deliberate?

Reply:
Yes, as discussed in the email thread